src/styles/blocks/card.css

```diff
@@ -0,0 +1,119 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 0 16px 16px;
```

Comment:
maybe we can set padding in this way - padding: 16px;

Reply:
> maybe we can set padding in this way - padding: 16px;

need to do it like this to have correct paddings